lib/cluster/rds.rb

```diff
@@ -147,10 +147,12 @@ def self.find_db_subnet_group
       end
     end
 
-    def self.sg_group_id
+    def self.sg_group_ids
       vpc = VPC.find_existing
       sg_finder = SecurityGroupFinder.new(vpc)
-      sg_finder.security_group_id_for('AWS-OpsWorks-Custom-Server')
+      [
+        sg_finder.security_group_id_for('OpsworksLayerSecurityGroupCommon')
```

Comment:
The RDS instance will be accessible to any instance that uses the common security group (i.e. all of them)